### ISSUE TITLE
Implement PUT in dynamic to post the response from an API 

### ISSUE THREAD

**Issue by GouravSardana** [IndianOpenSourceFoundation/dynamic-cli#83]
This feature will include the following things -

1. Once a user will enter dynamic -PUT, it will ask for a URL to put and the default URL would be 127.0.0.1:8000 and it will ask for the endpoint or you can put it directly in the URL itself
2. It will ask for headers then it will ask for payloads
3.  Optional param - User-Agent
4. It will build on top of CURL.
5. The output will get in JSON with proper syntax highlighter.

Example -
- dynamic -PUT
- Enter the URL with endpoint - [press enter - default would be 127.0.0.1:800]
- Headers - Enter headers
- User Agent [Optional]
- Codes => 200 (OK) or 204 (No Content). 404 (Not Found), if ID not found or invalid.
Posting (loading emoji)

```

{
json output
}

```

**Comment by guptaaastha** [comment]
Hi @GouravSardana can I take up this issue?

**Comment by GouravSardana** [comment]
Sure @guptaaastha . Please Go ahead

**Comment by guptaaastha** [comment]
@GouravSardana The CLI does not support windows OS and therefore will not be able to take up this issue at the moment. Please un-assign me. I'll also raise an issue on the repository for the same.

**Comment by pranavbaburaj** [comment]
> @GouravSardana The CLI does not support Windows OS and therefore will not be able to take up this issue at the moment. Please un-assign me. I'll also raise an issue on the repository for the same.

Yes @guptaaastha, the cli is not supporting windows right now due to the `termor` package used in the application. Some modules used are not yet implemented for windows. We are searching for a way to solve this issue. All contributions are appreciated.

**Comment by subhangi2731** [comment]
@pranavbaburaj can i work on this

**Comment by subhangi2731** [comment]
@GouravSardana please assign this issue to me

**Comment by pranavbaburaj** [comment]
> @pranavbaburaj can I work on this

Yeah, you can. But, the question is how 😐 ?

**Comment by Abhinavcode13** [comment]
is this issue open?
